Src/BootCamp.Chapter/Gambling/GameComponents/Deck.cs

```diff
@@ -39,9 +39,14 @@ public Card DrawAt(int index)
 
         private void ValidateDeckSize(int index)
         {
-            if (index >= _cards.Count)
+            
+            if (_cards.Count == 0)
```

Comment:
This one is not needed, only the condition below. If you have 0 cards, then index will be equal, which will throw.

Reply:
i wanted a different Error message. one is for OutOfCardsException(); when there are no cards in the deck. the other IndexOutOfRangeException().